### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by VivekRaaZ**
### **_Project requirements_**
________________

 Build the middle section of the mobile website. We included a screenshot of Template 1 below for reference, but you should stick to the template that you chose. This sections of the mobile website should include:
  - "Works" section✅
  - The "About myself" section (placed after the projects and before the contact form): About myself✅

- Make sure the content of the "About myself" section is aligned with flexbox.✅
- Use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.✅
- Please use ul and li for categories in "Work" section. Html, CSS, and JavaScript should be list items.✅
- Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.✅
- Implement the button interactions (enable, hover, etc.).✅
 The website should be responsive, following these breakpoints:
  - Minimum mobile screen size should be 375px✅
  - Minimum desktop screen size should be 768px✅

Hope you like it! 😃👍

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @VivekRaaZ, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request



We have reviewed your second project milestone: Portfolio: finish mobile version. We noticed that you created a new repository for this milestone and that it is missing all the requirements from the first milestone of Portfolio: setup and mobile because that is in your separate repository.

We would like to remind you that all changes for this project should be made in one repository. This will help us keep track of your progress and ensure that you meet all the requirements for each milestone.

Please make sure to update your repository with all the necessary files and changes from the first milestone before submitting your final project.

If you have any questions or concerns, please do not hesitate to reach out to us.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -55,26 +124,211 @@ h1 {
...
+  font-size: 17px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  width: 120px;
+  height: 48px;
+  margin-top: 15px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
```
Comment (zeff96) [inline style.css:231]: - [ ]  Great working implementing the figma design, however your button is still missing hover, active and pressed status as per the project requirement.

The button (e.g., "Get my resume") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design)

**Review by zeff96 — CHANGES_REQUESTED** [review]
### Status: CHANGES REQUIRED ♻️ ♻️ ♻️ 
Hi @VivekRaaZ , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


almost

## Highlights
✅ Linters passing
✅ Descriptive pull request
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Refactoring your code to make structure flow more presentable

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Efiamotu-1 — APPROVED** [review]
Hi @VivekRaaZ , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**